style.css

```diff
@@ -494,11 +500,157 @@ h5 {
     font-weight: 500;
     line-height: 24px;
     letter-spacing: 0.2px;
-    padding: 0px 234px 0px 234px;
+    
+}
+.footer__secondary--container{
+    display: flex;
+    align-items: center;
+    width: 100%;
+    max-width: 1050px;
 }
 
+/* STYLE VIDEO SECTION */
+.video-section{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background: linear-gradient(180deg, rgba(0, 0, 0, 0.00) 14.58%, rgba(56, 56, 56, 0.84) 100%);
 
+}
+.container__video{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 80px 0px 80px 0px;
+    max-width: 880px;
+}
+/* STYLE CLIENT SAY SECTION */
+.client-say{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 80px 0px;
 
+}
+.container__client-say{
```

Comment:
do not duplicate styles. You have different containers with the same styles. Use one container class for all sections